test/test_network_constructors.jl

```diff
@@ -68,29 +68,32 @@ end
     c_sys14 = PSB.build_system(PSITestSystems, "c_sys14")
     c_sys14_dc = PSB.build_system(PSITestSystems, "c_sys14_dc")
     systems = [c_sys5, c_sys14, c_sys14_dc]
-    objfuncs = [GAEVF, GQEVF, GQEVF]
     constraint_keys = [
         PSI.ConstraintKey(FlowRateConstraint, PSY.Line, "lb"),
         PSI.ConstraintKey(FlowRateConstraint, PSY.Line, "ub"),
         PSI.ConstraintKey(CopperPlateBalanceConstraint, PSY.System),
-        PSI.ConstraintKey(NetworkFlowConstraint, PSY.Line),
     ]
-    PTDF_ref = IdDict{System, PTDF}(
+    PTDF_ref = IdDict{System, Any}(
```

Comment:
Why is this changed to Any?

Reply:
I was trying to reuse the code from the VirtualPTDF too to avoid code repetition.